### ISSUE TITLE
ERROR: AmazonXHR.list is called without file parameter when upload is_truncated is true

### ISSUE THREAD

**Issue by abuisine** [cinely/mule-uploader#73]
error raised here:
```
if(is_truncated === "true") {
    var part_marker = xml.getElementsByTagName("NextPartNumberMarker")[0].textContent;
    AmazonXHR.list(auth, key, upload_id, chunk_size, function(new_parts) {
        callback(parts.concat(new_parts));
    }, error_callback, part_marker);
} else {
    callback(parts);
}
```
whereas prototype is:
```
AmazonXHR.list = function(auth, file, key, upload_id, chunk_size, callback, error_callback, marker) {
```

**Comment by abuisine** [comment]
this bug is triggered when uploading large files (tested with 16GiB)
problem arises at the end of the upload, or resuming such failed upload

**Comment by abuisine** [comment]
solved by 24863a2f86429353ef729640ab25b41a73fff543
